### PR TITLE
bump fmt/7.1.3

### PR DESCRIPTION
**Pull request by prince-chrismc**
Specify library name and version:  **fmt/7.1.3**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by danimtb** [comment]
Looks like we are experiencing some issues with one of our Windows agents and this PR got hit. Sorry for the inconvenience

**Review by Croydon — DISMISSED** [review]

**Review by madebr — APPROVED** [review]

**Review by ericLemanissier — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

File: recipes/fmt/all/conanfile.py [hunk 1]
```diff
@@ -38,6 +38,9 @@ def configure(self):
         elif self.options.shared:
             del self.options.fPIC
 
+        if self.settings.compiler == "Visual Studio" and self.options.get_safe("shared", False) and "MT" in self.settings.compiler.runtime:
+            raise ConanInvalidConfiguration("Visual Studio build for shared library with MT runtime is not supported")
```
Comment (madebr) [inline recipes/fmt/all/conanfile.py:42]:
```suggestion
            del self.options.fPIC
            if self.settings.compiler == "Visual Studio" and "MT" in self.settings.compiler.runtime:
                raise ConanInvalidConfiguration("Visual Studio build for shared library with MT runtime is not supported")
```

**Review by madebr — APPROVED** [review]

**Review by danimtb — APPROVED** [review]

**Review by ericLemanissier — APPROVED** [review]

**Comment by prince-chrismc** [comment]
CI status was inprogress still

**Review by theirix — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Review by danimtb — APPROVED** [review]

**Comment by prince-chrismc** [comment]
CCI still marked in progress

**Comment by prince-chrismc** [comment]
Can someone please look into why CCI is stuck? 🙏

**Review by intelligide — APPROVED** [review]

**Comment by intelligide** [comment]
@uilianries @danimtb CCI is stuck :sweat_smile:

**Comment by prince-chrismc** [comment]
6 approvals but no merge xD usually the opposite!

If someone gets the chance to see why the bots stuck that would be appreciated!

**Comment by danimtb** [comment]
For some reason, the status is not updated. Merging this manually

**Pull request merged**